### ISSUE TITLE
Prediction results of pre-trained weights

### ISSUE THREAD

**Issue by BenPinhasov** [HongguLiu/MesoNet.Pytorch#8]
Hello,

I use this implementation for mesoNet with pre-trained weights that you provided.
When I test it on real video from FF++ dataset, the accuracy of the model is very low. It predict the videos (and the frames from the video)  most of the time as fake.
When I test in on fake video from the same dataset the accuracy near to accuracy that declared in the original paper.
When I combined the real and fake datasets the accuracy 50%
I did something wrong with my tests?
